azure-quantum/azure/quantum/optimization/solvers.py

```diff
@@ -586,9 +586,9 @@ def __init__(
         beta: Optional[RangeSchedule] = None,
         steps_per_walker: Optional[int] = None,
     ):
-        """The constructor of Population Annealing Search solver.
+        """The constructor of  Substochastic Monte Carlo solver.
```

Comment:
This is a bit of a nit but there are two spaces here.
```suggestion
        """The constructor of Substochastic Monte Carlo solver.
```